github/config.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/github/hub/Godeps/_workspace/src/github.com/howeyc/gopass"
 	"github.com/github/hub/ui"
 	"github.com/github/hub/utils"
+	"github.com/github/hub/Godeps/_workspace/src/github.com/cep21/xdgbasedir"
```

Comment:
Instead of pulling in a new dependency, couldn't we simply read the value of `XDG_CONFIG_HOME` environment variable and default to `$HOME/.config` if not set?